recipes/nettle/3.6/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set version = "3.6" %}
+{% set sha256 = "d24c0d0f2abffbc8f4f34dcf114b0f131ec3774895f3555922fe2f40f3d5e3f1" %}
+
+package:
+  name: nettle-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://www.lysator.liu.se/~nisse/nettle/
+  license: GPL2
+  summary: A low-level cryptographic library
+
+build:
+  number: 5
```

Comment:
When we bump a version number, builds should be set to `0` (the first build of the new version)